src/flag_gems/ops/isclose.py

```diff
@@ -0,0 +1,134 @@
+import logging
+
+import torch
+import triton
+import triton.language as tl
+
+from ..utils import pointwise_dynamic
+from .all import all
+
+
+@pointwise_dynamic(is_tensor=[True, True, False, False], output_dtypes=[torch.bool])
+@triton.jit
+def isclose_func(x, y, rtol, atol):
+    x_fp = x.to(tl.float32)
+    y_fp = y.to(tl.float32)
```

Comment:
the operands should be compared in their original precision in case of no type promotion, right?